pkg/cmd/pulumi/org_search.go

```diff
@@ -21,7 +21,7 @@ import (
 	"io"
 	"os"
 
-	"github.com/hashicorp/go-multierror"
+	"github.com/goccy/go-json"
```

Comment:
Why is this necessary over the regular `import "encoding/json"`?

Reply:
I expect it's very much not, and it likely another copilot hallucination. Let me fix this up.